bagit.py

```diff
@@ -784,6 +784,15 @@ def _make_tagmanifest_file(alg, bag_dir):
         for digest, filename in checksums:
             tagmanifest.write('%s %s\n' % (digest, filename))
 
+def _find_tag_files(bag_dir):
+    for dirName, subdirList, fileList in os.walk(bag_dir):
+        if not re.match(r'.*data$', dirName):
+            for filename in fileList:
+                if re.match(r'^tagmanifest-.+\.txt$', filename):
```

Comment:
Rather than using a file regex, this could be faster and potentially easier to read as `if filename.startswith('tagmanifest-')`